btr-api/src/btr_api/models/person.py

```diff
@@ -0,0 +1,53 @@
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy import Column, String, DateTime, Boolean
+
+import uuid
+
+from .base import BtrModelBase
+from .db import db
+
+
+class Person(db.Model, BtrModelBase):
+    """It holds details of person that are public."""
+
+    __tablename__ = "persons"
+
+    uuid = Column(UUID(as_uuid=True), default=uuid.uuid4)  # used as external reference
+
+    full_name = Column(String(300), index=True, nullable=True)
+    family_name = Column(String(100), nullable=True)
+    given_name = Column(String(100), nullable=True)
+    patronymic_name = Column(String(100), nullable=True)
+
+    date_of_birth = Column(DateTime(timezone=True))
+
+    is_permanent_resident = Column(Boolean(), default=False)
+    is_canadian_citizen = Column(Boolean(), default=False)
+    is_canadian_tax_resident = Column(Boolean(), default=False)
```

Comment:
if we don't add submission_id to the base model I think it should be added here so we don't lose the submission link to each record

Reply:
Updated relationships as we discussed offline.